src/shared/scrapers/KOR/index.js

```diff
@@ -0,0 +1,88 @@
+import * as fetch from '../../lib/fetch/index.js';
+import * as parse from '../../lib/parse.js';
+
+const scraper = {
+  country: 'KOR',
+  aggregate: 'state', // Special cities have equal status to states.
+  type: 'table',
+  url: 'http://ncov.mohw.go.kr/en/bdBoardList.do?brdId=16&brdGubun=162&dataGubun=&ncvContSeq=&contSeq=&board_id=',
+  sources: [
+    {
+      url: 'http://ncov.mohw.go.kr/en/bdBoardList.do?brdId=16&brdGubun=162&dataGubun=&ncvContSeq=&contSeq=&board_id=',
+      name: 'Ministry of Health and Welfare'
+    }
+  ],
+  maintainers: [
+    {
+      name: 'Jacob McGowan',
+      github: 'jacobmcgowan'
+    }
+  ],
+  _fieldPositions: {
+    cases: 3,
+    recovered: 4,
+    deaths: 5
+  },
+  _provinceNames: {
+    'Chungcheongbuk-do': 'North Chungcheong',
+    'Chungcheongnam-do': 'South Chungcheong',
+    'Gangwon-do': 'Gangwon',
+    'Gyeonggi-do': 'Gyeonggi',
+    'Gyeongsangbuk-do': 'North Gyeongsang',
+    'Gyeongsangnam-do': 'South Gyeongsang',
+    'Jeollabuk-do': 'North Jeolla',
+    'Jeollanam-do': 'South Jeolla',
+    Lazaretto: 'Quarantine'
+  },
+  _population: {
+    'Chungcheongbuk-do': 1588633,
+    'Chungcheongnam-do': 2064665,
+    'Gangwon-do': 1549780,
+    'Gyeonggi-do': 12239862,
+    'Gyeongsangbuk-do': 2739179,
+    'Gyeongsangnam-do': 3374725,
+    'Jeollabuk-do': 1895882,
+    'Jeollanam-do': 1938136,
+    Jeju: 583284,
+    Seoul: 9904312,
+    Busan: 3448737,
+    Daegu: 2466052,
+    Incheon: 2890451,
+    Gwangju: 1502881,
+    Daejeon: 1538394,
+    Ulsan: 1166615,
+    Sejong: 204088
+  },
+  /**
+   * Parses the common name of the province that is used in the feature.
+   * @param {string} officialEnglishName Official English name of the province.
+   * @returns {string} The common name of the province or the city name if a
+   * special city was given.
+   */
+  _parseProvince(officialEnglishName) {
+    return this._provinceNames[officialEnglishName] || officialEnglishName;
+  },
+  async scraper() {
+    const states = [];
+    const $ = await fetch.page(this.url);
+    const $table = $('table.num');
```

Comment:
Can you test to be sure you found the table? 

`if ($table.length === 0) { throw new Error('Table not found'); }`

Reply:
Can you also check each column header against your `fieldPositions` to be sure you throw an error if they change the table layout?